app.html

```diff
@@ -1246,6 +1337,10 @@
     </div>
     <br>
   </div>
+
+  <div id=connect_hid style='visibility:hidden;' class=control>
```

Comment:
Is there any way to make this just one "USB" button which automatically detects if it's a teensy or proffieboard?

Reply:
Yes, that is an idea I have been playing around with that idea as well. Did not do much to make a test version for this, but seems do-able, so I will give it a try.
My idea would be to show all available compatible devices (BLE and USB) for the user to select, without having to know if it is a Teensy Proffie connected via USB or BLE. Once the user selects a device, the appropriate configuration should be activated for the webapp.